doc/user/data/self_managed/unified_terraform_versions.yml

```diff
@@ -0,0 +1,16 @@
+columns:
+  - column: "Terraform version"
+  - column: "Notable changes"
+
+rows:
+
+- "Terraform version": |
+    v1.0.2
```

Comment:
It's actually probably v1.0.3 since v1.02 was from last week and we probably want the latest in main to be our GA.

https://github.com/MaterializeInc/materialize-terraform-self-managed/tags